### PR TITLE
[4.0] tooltip text alignment

### PR DESCRIPTION
**Pull request by brianteeman**
Simple PR adds clarity and improves readability

dont forget npm i

example on smart search

### before
image

### after
image

### CODE REVIEW

**Comment by ciar4n** [comment]
I have tested this item :white_check_mark: successfully on f792bff8585fe0fb748ca02fd86a4a00d56a3ee4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27554">issues.joomla.org/tracker/joomla-cms/27554</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on f792bff8585fe0fb748ca02fd86a4a00d56a3ee4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27554">issues.joomla.org/tracker/joomla-cms/27554</a>.</sub>

**Comment by SharkyKZ** [comment]
RTC.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27554">issues.joomla.org/tracker/joomla-cms/27554</a>.</sub>

**Comment by Quy** [comment]
For RTL, it should be right-aligned.

**Comment by infograf768** [comment]
Back to pending until rtl is taken care of to avoid another pr for mini stuff<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27554">issues.joomla.org/tracker/joomla-cms/27554</a>.</sub>

**Comment by infograf768** [comment]
Suggestion:
```
[...]
  z-index: $zindex-tooltip;

  [dir=ltr] & {
    text-align: left;
  }
  
  [dir=rtl] & {
    text-align: right;
  }
```

**Comment by brianteeman** [comment]
Thanks. I will update soon

**Comment by brianteeman** [comment]
updated for rtl

**Comment by infograf768** [comment]
As change is obvious, setting rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27554">issues.joomla.org/tracker/joomla-cms/27554</a>.</sub>

File: administrator/templates/atum/scss/blocks/_form.scss [hunk 1]
```diff
@@ -128,6 +128,14 @@ legend {
   background: $black;
   max-width: 100%;
   z-index: $zindex-tooltip;
+
+  [dir=ltr] & {
+  text-align: left;
+  }
+
+  [dir=rtl] & {
+  text-align: right;
```
Comment (infograf768) [inline administrator/templates/atum/scss/blocks/_form.scss]: @brianteeman 
needs 2 spaces before `text-align: right;`
same needs 2 spaces before `text-align: left;`
I leave the rtc

**Comment by brianteeman** [comment]
Done it - its hard to check this sort of thing from my phone

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**